src/System.Runtime.Handles/ref/System.Runtime.Handles.Manual.cs

```diff
@@ -6,26 +6,11 @@
 // ------------------------------------------------------------------------------
 
 
-namespace Microsoft.Win32.SafeHandles
-{
-    public partial class SafeWaitHandle
-    {
-        // Manually added because SafeHandleZeroOrMinusOneIsInvalid is removed.
-        public override bool IsInvalid {[System.Security.SecurityCriticalAttribute]get { return default(bool); } }
-    }
-}
-
 namespace System.Runtime.InteropServices
 {
     public partial class CriticalHandle
     {
         // Manually added because CriticalFinalizerObject.Finalize is removed.
```

Comment:
Since you're adding back CriticalFinalizerObject, will this be removed? (I guess in a subsequent PR?)